cmd/utils/flags.go

```diff
@@ -976,14 +976,14 @@ var (
 		Category: flags.RollupCategory,
 		Value:    true,
 	}
-	RollupSequencerEnableTxConditionalFlag = &cli.BoolFlag{
-		Name:     "rollup.sequencerenabletxconditional",
+	RollupSequencerTxConditionalEnabledFlag = &cli.BoolFlag{
+		Name:     "rollup.sequencertxconditionalenabled",
 		Usage:    "Serve the eth_sendRawTransactionConditional endpoint and apply the conditional constraints on mempool inclusion & block building",
 		Category: flags.RollupCategory,
 		Value:    false,
 	}
-	RollupSequencerTxConditionalRateLimitFlag = &cli.IntFlag{
-		Name:     "rollup.sequencertxconditionalratelimit",
+	RollupSequencerTxConditionalCostRateLimitFlag = &cli.IntFlag{
+		Name:     "rollup.sequencertxconditionalcostratelimit",
 		Usage:    "Maximum cost -- storage lookups -- allowed for conditional transactions in a given second",
```

Comment:
This isn't necessarily the number of storage lookups but I see why framing it that way makes it more simple to explain

Reply:
Yeah it seemed like the easiest way to describe this